source/cgal/surface_mesh.cc

```diff
@@ -0,0 +1,126 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+
+#include <deal.II/base/patterns.h>
+
+#include <deal.II/cgal/surface_mesh.h>
+
+#ifdef DEAL_II_WITH_CGAL
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace
+{
+  template <typename dealiiFace, typename Container, typename CGAL_Mesh>
+  void
+  add_facet(const dealiiFace &face,
+            const Container & deal2cgal,
```

Comment:
```suggestion
            const std::map<unsigned int, Vertex_index> & deal2cgal,
```

Could you be explicit in the variable what is translated here (vertex index).